draft-quic-atsss-reqs.mkd

```diff
@@ -138,19 +138,20 @@ informative:
     
 --- abstract
 
-This document briefly presents the Access Traffic Steering, Switching, and Splitting (ATSSS) service being specified within the 3rd Generation Partnership Project (3GPP). 
+This document briefly presents the Access Traffic Steering, Switching, and Splitting (ATSSS) service being specified within the 3rd Generation Partnership Project (3GPP).
 
 --- middle
 
 # Introduction {#intro}
 
-The 3rd Generation Partnership Project (3GPP) has described the Access Traffic Steering, Switching, and Splitting (ATSSS) service, used to direct TCP traffic between endpoints with multiple available paths, in their Release 16 {{TS23501}}. In Release 16, ATSSS uses the IETF MPTCP protocol {{RFC6824}}.
+The 3rd Generation Partnership Project (3GPP) has described the Access Traffic Steering, Switching, and Splitting (ATSSS) service, used to carry TCP traffic over multiple available paths, in their Release 16 {{TS23501}}. In Release 16, ATSSS only supports TCP traffic and relies upon two IETF protocols: MPTCP {{RFC6824}} and Convert Protocol {{I-D.ietf-tcpm-converters}}.
 
-As part of preparation for Release 17 studies, 3GPP has expressed an interest in other IETF protocols and protocol extensions that would remove some of the limits they experience with MPTCP. 
+As part of preparation for Release 17 studies, 3GPP has expressed an interest in other IETF protocols and protocol extensions that would enable ATSSS service of traffic not supported by the Convert Protocol and based on the use of MPTCP. To that aim, 
```

Comment:
```suggestion
As part of preparation for Release 17 studies, 3GPP has expressed an interest in other IETF protocols and protocol extensions that would enable ATSSS service of traffic not supported by the Convert Protocol nor based on the use of MPTCP. To that aim, 
```
I feel this is less ambiguous. Otherwise looks good!